cmake/install/install.cmake

```diff
@@ -11,20 +11,24 @@ include(GNUInstallDirs)
 # Library installs
 
 # Define the public headers for wiredtiger library to be used when installing the target.
```

Comment:
Should this comment line go with the following removed lines?

Reply:
Good point. I re-used the comment for the install invocation below since it still talked about installing the public wiredtiger headers. But this comment is still a tiny bit confusing as it talks about editing the wiredtiger target. So I reworded it to more appropriately match the `install` command :)